python/install

```diff
@@ -45,9 +46,10 @@ apt-get remove --purge -y \
         libncurses5-dev libncursesw5-dev \
         tk-dev git
 
-apt-get install -y --no-install-recommends \
+apt-get install -y \
         libssl1.0.0 zlib1g libbz2-1.0 \
         libsqlite3-0 libncurses5 \
         libncursesw5 libreadline7
 
-apt-get autoremove -y
\ No newline at end of file
+apt-get autoremove -y
+apt-get clean
```

Comment:
If you clean, the image will be smaller, but it may take longer to install dependencies from the requirements.apt file, right?